src/TokenHandler/AccountManager.mo

```diff
@@ -58,6 +62,17 @@ module {
     /// Total funds underway for consolidation.
     var underwayFunds : Nat = 0;
 
+    /// Total dust deposits (i.e. those that are less than the fee and insufficient for consolidation).
+    var dustDeposits : Nat = 0;
```

Comment:
Suggesting to rename it to `dustFunds` to be consistent with naming of other variables.